CRM/Utils/Address/BatchUpdate.php

```diff
@@ -234,6 +239,15 @@ public function processContacts($processGeocode, $parseStreetAddress) {
           $unparseableContactAddress[] = " Contact ID: " . $dao->id . " <a href =\"$url\"> " . $dao->street_address . " </a> ";
           // reset element values.
           $parsedFields = array_fill_keys(array_keys($parsedFields), '');
+          /**
```

Comment:
These lines should be moved to L219 in an else code block. This is because this part of the code is about street addressing parsing not geocoding